demos/using-dev-build/README.md

```diff
@@ -1,16 +1,68 @@
 # Integrating with Internet Identity
 
-This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors).
+This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library. This is a standalone project that you can copy to your own project.
 
-## Running the tests
+## Usage
 
 The following commands will start a replica, install the development Internet Identity canister, and run the test suite:
 
 ```bash
-$ dfx start # in a different terminal
-$ dfx deploy --no-wallet --argument '(null)'
+$ dfx start --background --clean
 $ npm ci
-$ npm run test
+$ dfx deploy --no-wallet --argument '(null)'
```

Comment:
Maybe adding `npm start` right after this?

Without explaining yet why, but so that they can see it in action.

And then a comment of "visit localhost:8080"

Reply:
The `npm run start` can be a bit confusing since at that point the frontend is served twice, I'd rather do things progressively!